library/psa_crypto.c

```diff
@@ -4554,7 +4692,9 @@ static psa_status_t psa_key_derivation_internal(
 
     return( PSA_SUCCESS );
 }
+#endif /* PSA_PRE_1_0_KEY_DERIVATION */
 
+#if defined(PSA_PRE_1_0_KEY_DERIVATION)
```

Comment:
Is this conditional and the previous #endif redundant? The both seem to have the same macro conditional

Reply:
This redundancy is intentional, makes it easier to remove/refactor these functions individually.